.travis/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# This script was adapted from work by Keith James (keithj). The original source
+# can be found as part of the wtsi-npg/data_handling project here:
+#
+#   https://github.com/wtsi-npg/data_handling
+
+set -e -x
+
+sudo apt-get install libgd2-xpm-dev # For npg_tracking
+sudo apt-get install liblzma-dev # For npg_qc
+sudo apt-get install --yes nodejs
+
+# CPAN
+cpanm --quiet --notest Alien::Tidyp # For npg_tracking
+cpanm --quiet --notest Module::Build
+cpanm --quiet --notest Config::Auto # For npg_qc
+cpanm --quiet --notest Readonly
+
+# WTSI NPG Perl repo dependencies
+cd /tmp
+git clone https://github.com/wtsi-npg/ml_warehouse.git ml_warehouse.git
+git clone https://github.com/wtsi-npg/npg_ml_warehouse.git npg_ml_warehouse.git
+git clone https://github.com/wtsi-npg/npg_tracking.git npg_tracking.git
+git clone https://github.com/wtsi-npg/npg_seq_common.git npg_seq_common.git
+
+cd /tmp/ml_warehouse.git    ; git checkout "${DNAP_WAREHOUSE_VERSION}"
+cd /tmp/npg_ml_warehouse.git; git checkout "${NPG_ML_WAREHOUSE_VERSION}"
+cd /tmp/npg_tracking.git    ; git checkout "${NPG_TRACKING_VERSION}"
+cd /tmp/npg_seq_common.git  ; git checkout "${NPG_SEQ_COMMON_VERSION}"
+
+# Fix seq_common
+rm /tmp/npg_seq_common.git/t/bin/aligners/bwa/bwa-0.5.8c/bwa
+cp /tmp/npg_seq_common.git/t/bin/bwa /tmp/npg_seq_common.git/t/bin/aligners/bwa/bwa-0.5.8c/bwa
+
+rm -r /tmp/npg_seq_common.git/t/data/references/Homo_sapiens/default
+cp -R /tmp/npg_seq_common.git/t/data/references/Homo_sapiens/NCBI36 /tmp/npg_seq_common.git/t/data/references/Homo_sapiens/default
+
+# These are cruft, apparently
+rm -r /tmp/npg_seq_common.git/t/data/sequence/references2
+
+repos="${TRAVIS_BUILD_DIR} ${TRAVIS_BUILD_DIR}/npg_qc_viewer /tmp/ml_warehouse.git /tmp/npg_ml_warehouse.git /tmp/npg_tracking.git /tmp/npg_seq_common.git"
+
+# Install CPAN dependencies. The src libs are on PERL5LIB because of
+# circular dependencies. The blibs are on PERL5LIB because the package
+# version, which cpanm requires, is inserted at build time. They must
+# be before the libs for cpanm to pick them up in preference.
+
+for repo in $repos
+do
+  export PERL5LIB=$repo/blib/lib:$PERL5LIB:$repo/lib
+done
+
+for repo in $repos
```

Comment:
For this and previous comment.

ml_warehouse: recommends npg_qc::autoqc::role::rpt_key
npg_seq_common: 
  ! Couldn't find module or a distribution npg_qc::Schema
  ! Couldn't find module or a distribution npg_qc::autoqc::results::bam_flagstats
  ! Installing the dependencies failed: Module 'npg_qc::Schema' is not installed, Module 'npg_qc::autoqc::results::bam_flagstats' is not installed
  ! Bailing out the installation for npg_seq_common-0.0-bf93337-dirty.
  requires:
    !  npg_qc::Schema is not installed
    !  npg_qc::autoqc::results::bam_flagstats is not installed

So I was bootstrapping. But I agree it will be better to have npg_qc completely apart.